server/interceptor/interceptor.go

```diff
@@ -155,7 +155,7 @@ func (tp *trillianProcessor) Before(ctx context.Context, req interface{}) (conte
 
 	if info.getTree {
 		tree, err := trees.GetTree(
-			ctx, tp.parent.admin, info.treeID, trees.NewGetOpts(trees.Admin, info.readonly, info.treeTypes...))
+			ctx, tp.parent.admin, info.treeID, trees.NewGetOpts(trees.Admin, info.treeTypes...))
```

Comment:
Is the `readonly` flag in `info` needed then? Maybe remove it if not.

Reply:
Yes, because it's passed to the quota API.

Reply:
Ah, ok.